ui/js/values.js

```diff
@@ -124,6 +124,42 @@ treeherder.value("phTimeRanges", [
 
 treeherder.value("phDefaultTimeRangeValue", 1209600);
 
+treeherder.value("phDatePoints", [
```

Comment:
`phDatePoints` seems somewhat ambiguous to me. `phComparisonDate` sounds a bit clearer?